job.go

```diff
@@ -221,6 +224,22 @@ func (j *Job) updateConnections() {
 				})
 				continue
 			}
+			if strings.HasPrefix(conn, "rds-postgres://") {
+				// reuse postgres SQLDriver by stripping rds- from connexion URL after building the RDS
+				// authentication token
+				conn = strings.TrimPrefix(conn, "rds-")
+				// FIXME - parsing twice the conn url to extract host & username
+				u, err := url.Parse(conn)
+				region := os.Getenv("AWS_REGION")
```

Comment:
What appens if AWS_REGION env is not set ?

Reply:
Then the default region will be used, and the generated token will be wrong, the user will get a generic postgresql authentication error.